apps/opik-backend/src/main/java/com/comet/opik/utils/DurationUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.comet.opik.utils;
+
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+
+import java.time.Duration;
+import java.time.Instant;
+
+@UtilityClass
+public class DurationUtils {
```

Comment:
This should be nuked as well.

Reply:
Done in https://github.com/comet-ml/opik/pull/883/commits/00e0752e8c158086e97533a30a7a24620972212d